docs/02.API-REFERENCE.md

```diff
@@ -1092,6 +1093,7 @@ jerry_get_global_object (void);
 - [jerry_release_value](#jerry_release_value)
 - [jerry_define_own_property](#jerry_define_own_property)
 
+
```

Comment:
This new line is unnecessary.

Reply:
IMHO it is necessary. We put two spaces before top level headers (`#`) to make it more readable in raw format. It helps to notice where a new chapter starts. This style is used in the whole MD file.